sagemaker-python-sdk/mxnet_gluon_mnist/mxnet_mnist_with_gluon_1.3.ipynb

```diff
@@ -0,0 +1,1139 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Local MNIST Training with MXNet and Gluon\n",
+    "\n",
+    "### Pre-requisites\n",
+    "\n",
+    "This notebook shows how to use the SageMaker Python SDK to run your code in a local container before deploying to SageMaker's managed training or hosting environments.  This can speed up iterative testing and debugging while using the same familiar Python SDK interface.  Just change your estimator's `train_instance_type` to `local`.  You could also use `local_gpu` if you're using an ml.p2 or ml.p3 notebook instance, but then you'll need to set `train_instance_count=1` since distributed, local, GPU training is not yet supported.\n",
```

Comment:
I'd put this paragraph under the title rather than "prerequisites". Also, mention MXNet and Gluon in here.